docs/index.md

```diff
@@ -248,6 +248,37 @@ To facilitate this workflow, the issuer can include a link to help the user down
   ]
 }
 ```
+#### on Android
+When transferring a SMART Health Card from one app to another on Android, use:
+
+```java
+Intent intent = new Intent();
+intent.setAction(Intent.ACTION_SEND);
+intent.setType("application/smart-health-card");
+intent.putExtra(Intent.EXTRA_STREAM, uri);
+intent.addFlags(Intent.FLAG_GRANT_READ_URI_PERMISSION);
+intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
+startActivityForResult(Intent.createChooser(intent, ...
```

Comment:
Does this pass a `uri` only, or a complete `.smart-health-card` JSON file?

Reply:
(The `uri` has a limited length and can only convey a single chunk of a single JWS, but the full JSON file can a full JWS and even more than one JWS.)